app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  validates :name, presence: true
```

Comment:
[OPTIONAL] Consider changing name to username as given in the specs:

![Username requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/building_with_active_record/userValidation.png)